src/components/Tiles/TilesGrid.js

```diff
@@ -0,0 +1,24 @@
+import ApiProvider from '../../providers/ApiProvider';
+import { render } from '../../shared/dom';
+import { MovieTile } from './MovieTile';
+
+export const TilesGrid = ({ renderOn, movies }) => {
+    const template = `
+        <div class='tiles-grid-container'>
```

Comment:
I think that we should stick to the BEM methodology, and just name it, `gridContainer`